class/frontend-dashboard.php

```diff
@@ -27,13 +27,11 @@ function shortcode( $atts ) {
             <?php //echo $custom_css = wpuf_get_option( 'custom_css', 'wpuf_general' ); ?>
         </style>
         <?php
-
-        extract( shortcode_atts( array('post_type' => 'post'), $atts ) );
-
+        extract( shortcode_atts( array( 'form_id'=>'off', 'post_type' => 'default', 'category' =>'off', 'featured_image' => 'default', 'meta' => 'off', 'excerpt' =>'off'), $atts ) );

```

Comment:
Instead of extracting the array, as the arguments are getting larger, we should pass the whole array to `post_listing` function. Increasing number of function parameters are not manageable.